src/utilsV2.py

```diff
@@ -50,10 +49,10 @@ def load_image2(path, height=None, width=None):
 # Render the generated image given a tensorflow session and a variable image (x)
 def render_img(session, x, save=False, out_path=None):
     shape = x.get_shape().as_list()
-    img = np.clip(session.run(x), 0, 1)
+    img = (np.clip(session.run(x), 0, 1) * 255).astype(np.uint8)
 
     if save:
-        Image.fromarray(np.reshape(img, shape[1:])).save(out_path)
+        Image.fromarray(np.reshape(img, shape[1:])).resize((1200, 1200)).save(out_path)
```

Comment:
Why are we resizing to 1200x1200? If this is arbitrary, can we make this configurable via flag arg?